extensions/pkg/controller/controlplane/genericactuator/actuator.go

```diff
@@ -204,7 +204,7 @@ func (a *actuator) reconcileControlPlaneExposure(
 	// Apply control plane exposure chart
 	a.logger.Info("Applying control plane exposure chart", "controlplaneexposure", kutil.ObjectName(cp), "values", values)
 	version := cluster.Shoot.Spec.Kubernetes.Version
-	if err := a.controlPlaneExposureChart.Apply(ctx, a.chartApplier, cp.Namespace, a.imageVector, a.gardenerClientset.Version(), version, values); err != nil {
+	if err := a.controlPlaneExposureChart.Apply(ctx, a.chartApplier, a.client, cp.Namespace, a.imageVector, a.gardenerClientset.Version(), version, values); err != nil {
```

Comment:
You need to add a release note that it is a breaking change.

Reply:
Done